source/extensions/dynamic_modules/abi.h

```diff
@@ -127,6 +130,17 @@ typedef char* envoy_dynamic_module_type_buffer_module_ptr;
  */
 typedef char* envoy_dynamic_module_type_buffer_envoy_ptr;
 
+/**
+ * envoy_dynamic_module_type_module_http_header represents a key-value pair of an HTTP header owned
+ * by the module.
+ */
+typedef struct {
+  envoy_dynamic_module_type_buffer_module_ptr key_ptr;
+  size_t key_length;
+  envoy_dynamic_module_type_buffer_module_ptr value_ptr;
+  size_t value_length;
+} envoy_dynamic_module_type_module_http_header;
+
 /**
  * envoy_dynamic_module_type_Header represents a key-value pair of an HTTP header owned by Envoy's
```

Comment:
while there, could you fix this comment...
```suggestion
 * envoy_dynamic_module_type_http_header represents a key-value pair of an HTTP header owned by Envoy's
```